cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -185,124 +186,228 @@ func (pfct preflightCheckTest) Check() (warning, errorList []error) {
 	return
 }
 
-func TestRunInitNodeChecks(t *testing.T) {
+func TestFileExistingCheck(t *testing.T) {
+	f, err := os.CreateTemp("/tmp", "exist")
```

Comment:
```suggestion
	f, err := os.CreateTemp("", "file-exist-check")
```

empty string means use the OS default temp directory.